### PR TITLE
INFRA: Initialize Unit Test Project

### LINKED ISSUE #4: INFRA: Initialize Unit Test Project

### PR DESCRIPTION
**Pull request by rcruzfreelance**
closes #4

### CODE REVIEW

File: STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace STX.EFxceptions.PostgreSQL.Base.Tests.Unit
```
Comment (glhays) [inline STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs:5]: Add copyright header.

File: STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+{
+    public class DeletMe
```
Comment (glhays) [inline STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs]: Fix spelling.

**Comment by glhays** [comment]
@rcruzfreelance Ricardo, I noticed in this pr you had to point the icon locale to the images folder.
This was my bad by not putting a copy in the root of Resources where it should be so the template @cjdutoit setup for creating new repositories points to by default. Do you mind correcting that?

File: STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs (hunk 1 above)
Comment (rcruzfreelance) [reply STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs:5]: done

File: STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs (hunk 2 above)
Comment (rcruzfreelance) [reply STX.EFxceptions.PostgreSQL.Base.Tests.Unit/DeleteMe.cs]: done

**Review by glhays — APPROVED** [review]

**Pull request merged**